spec/cli_spec.rb

```diff
@@ -73,6 +73,80 @@
 - + - + -
 6 | 7 | 8
 
+GAMEOUTPUT
+
+    end
+  end
+
+  context "a game that ends in a draw" do
+    let(:input) { StringIO.new("0\n1\n4\n2\n5\n3\n6\n8\n7") }
```

Comment:
In all these tests we create an `input` with different parameters, it might be better to extract out this `input` into the top level of the tests (a `let` block in the outermost `describe`) that takes an array of moves and `join`s them with a `\n`s. This way your tests are written in the language of the domain of noughts and crosses and less in the language of the computer.